main.go

```diff
@@ -256,22 +256,24 @@ func (cfg Config) validate() error {
 }
 
 const notConnected = `Connected Apple Developer Portal Account not found.
-Most likely because there is no Apple Developer Portal Account connected to the build, or the build is running locally.
+Most likely because there is no Apple Developer Portal Account connected to the build.
 Read more: https://devcenter.bitrise.io/getting-started/configuring-bitrise-steps-that-require-apple-developer-account-data/`
 
 func handleSessionDataError(err error) {
 	if err == nil {
 		return
 	}
 
-	if networkErr, ok := err.(devportalservice.NetworkError); ok && networkErr.Status == http.StatusNotFound {
+	if networkErr, ok := err.(devportalservice.NetworkError); ok && networkErr.Status == http.StatusUnauthorized {
 		log.Debugf("")
-		log.Debugf("%s", notConnected)
-	} else {
-		fmt.Println()
-		log.Errorf("Failed to activate Bitrise Apple Developer Portal connection: %s", err)
-		log.Warnf("Read more: https://devcenter.bitrise.io/getting-started/configuring-bitrise-steps-that-require-apple-developer-account-data/")
+		log.Debugf("%s", "Unauthorized to query Connected Apple Developer Portal Account. This likely happens with a public app's PR build by design, to protect secrets.")
```

Comment:
What about: `This happens, by design, with a public app's PR build, to protect secrets.`

Reply:
updated